<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoxelClipPlaneWidget</class>
 <widget class="QWidget" name="VoxelClipPlaneWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VoxelClipPlaneWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item row="12" column="2">
    <widget class="QSpinBox" name="spb_invZClip">
     <property name="maximum">
      <number>30000</number>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QSlider" name="hsld_yClip">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QSlider" name="hsld_invYClip">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QSpinBox" name="spb_zClip">
     <property name="maximum">
      <number>30000</number>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QSpinBox" name="spb_invXClip">
     <property name="maximum">
      <number>30000</number>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSlider" name="hsld_xClip">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QSlider" name="hsld_zClip">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QSpinBox" name="spb_xClip">
     <property name="maximum">
      <number>30000</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QSpinBox" name="spb_yClip">
     <property name="maximum">
      <number>30000</number>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QSlider" name="hsld_invZClip">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QSpinBox" name="spb_invYClip">
     <property name="maximum">
      <number>30000</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSlider" name="hsld_invXClip">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="ckb_xClip">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="lb_voxel">
     <property name="text">
      <string>Voxel:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QComboBox" name="cbb_voxel"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="lb_xClip">
     <property name="text">
      <string>X-Clip:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="lb_yClip">
     <property name="text">
      <string>Y-Clip:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QLabel" name="lb_zClip">
     <property name="text">
      <string>Z-Clip:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="ckb_invXClip">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="ckb_yClip">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QCheckBox" name="ckb_invYClip">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QCheckBox" name="ckb_zClip">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QCheckBox" name="ckb_invZClip">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="TitleBar" name="titleBar" native="true"/>
   </item>
   <item row="13" column="0" colspan="3">
    <spacer name="vspr_bottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>219</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TitleBar</class>
   <extends>QWidget</extends>
   <header>titlebar.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
